site/sfguides/src/machine_learning_with_snowpark_python/machine_learning_with_snowpark_python.md

```diff
@@ -0,0 +1,429 @@
+- Snowflake Ingest Capabilities & Performance (loading ~94M records in roughly 30s)
+
+In this section of the demo, we will utilize Snowpark's Python client-side Dataframe API to build an ELT pipeline. We will extract the data from the source system (s3), load it into snowflake and add transformations to clean the data before analysis.
+
+Open up the [`01_Data_Engineering`](https://github.com/Snowflake-Labs/vhol-citibike-ml-snowpark-python/blob/main/01_Data_Engineering.ipynb) notebook and execute the cells. This notebook will:
+
+Now that we've done our initial engineering and loaded a sample of data, let's create some bulk ELT processes using the exported modules from the Data Engineering notebook.
+
+Open up the [`01_Ingest`](https://github.com/Snowflake-Labs/vhol-citibike-ml-snowpark-python/blob/main/01_Ingest.ipynb) notebook and walk through the cells to see how to incrementally and bulk load data from S3 into Snowflake tables.
+
+Run the last few cells of the notebook to bulk-load ~94M records into your Snowflake table in only about 30 seconds.
+
+<!-- ------------------------ -->
+## Data Marketplace
+Duration: 5
+
+**Persona**: Data Engineer
+
+**What You'll Do**: Use the Snowflake Data Marketplace to subscribe to weather data from Weather Source.
+
+**Snowflake Features**:
+- [Snowflake Data Marketplace](https://www.snowflake.com/data-marketplace/)
+
+Weather Source is a leading provider of global weather and climate data and the OnPoint Product Suite provides businesses with the necessary weather and climate data to quickly generate meaningful and actionable insights for a wide range of use cases across industries.
+
+Weather Source powers a majority of Fortune companies who use their data to quantify the impact of weather and climate on various KPIs including footfall traffic, product sales and demand, supply chain and logistics, advertising and more.
+
+Open up the [02_Data_Marketplace](https://github.com/Snowflake-Labs/vhol-citibike-ml-snowpark-python/blob/main/02_Data_Marketplace.ipynb) notebook and follow the instructions to add the Snowpark Hands-on-Lab Weather Source dataset to your Snowflake account.
+
+1. Go the Snowflake UI, and select `Marketplace`
+2. Search for `snowpark` and click on the Snowpark for Python - Hands-on Lab - Weather Data tile
+![](assets/weathersource_tile.png)
+3. Click `Get Data` and add the dataset to your Snowflake account
+![](assets/weathersource_getdata.png)
+
+<!-- ------------------------ -->
+## Data Science Exploration and Modeling
+Duration: 20
+
+**Persona**: Data Scientist
+
+**What You'll Do**: 
+- Query daily trip data in Snowflake using Snowpark Python and visualize time-series data
+- Train a preliminary PyTorch TabNet model to predict daily trip counts
+- Augment your model feature trips data with Holiday reference data, along with weather data available in the [Snowflake Data Marketplace](https://www.snowflake.com/data-marketplace/?_sft_dataset-category=weather)
```

Comment:
@sfc-gh-cbaechtold Pl change Snowflake Data Marketplace to Snowflake Marketplace.